docs/content/doc/advanced/customizing-gitea.en-us.md

```diff
@@ -96,6 +96,31 @@ Google Analytics, Matomo (previously Piwik), and other analytics services can be
 
 Place custom files in corresponding sub-folder under `custom/options`.
 
+**NOTE:** The files should not have a file extension, e.g. `Labels` rather than `Labels.txt`
+
+### gitignores
+
+To add custom .gitignore, add a file with existing [.gitignore rules](https://git-scm.com/docs/gitignore) in it to `custom/options/gitignore`
+
+### Labels
+
+To add a custom label set, add a file that follows [this](https://github.com/go-gitea/gitea/blob/master/options/label/Default) format to `custom/options/label`  
+`#hex-color label name ; label description`
+
+### Licenses
+
+To add a custom license, add a file with the license text to `custom/options/license`
+
+### Locales
+
+Locales are managed via our [crowdin](https://crowdin.com/project/gitea)  
+To override a locale, add the appropriate file to `custom/options/locale`. The list of included files can be found [here](https://github.com/go-gitea/gitea/tree/master/options/locale) to use as examples.
```

Comment:
Hmm, again "here" is a bad link. Can we reword this? 

```markdown
You can override a locale by placing an altered locale file in `custom/options/locale`. Gitea's default locale files can be found in  the [`options/locale`](https://github.com/go-gitea/gitea/tree/master/options/locale) source folder and these should be used as examples for your changes.
```

Do you know if Gitea will fallback to its original files if you only want to change a few things? (I hope it does) in either case we should make it clear what happens.

Hmm, also if you add a new locale will it add it? (I guess it won't - but it probably should)

Reply:
I'll run some tests to make sure and then update with my findings.

Reply:
> Do you know if Gitea will fallback to its original files if you only want to change a few things?

Gitea will completely override the locale with the supplied one. Of course, you could still just copy the one from source and then tweak it a bit.

> Hmm, also if you add a new locale will it add it?

Yes! As long as you add it to the `[i18n]` section of `app.ini` Gitea will use it.

Reply:
☝️ Same goes for files in "public" directory and licenses too

Reply:
To clarify, do I need to change some wording regarding those directories/files?  
Perhaps just a note that all custom files will override their original source file?

Reply:
>  Perhaps just a note that all custom files will override their original source file?
There is a note under templates in existing template customization docs, but I agree with @zeripath that the changes need to be made in this section too (due to modification of app.ini), and the changes with regards to sentence structure.

Reply:
Added some more info.